benchopt/cli/helpers.py

```diff
@@ -40,6 +45,134 @@ def clean(benchmark, token=None, filename=None):
     rm_folder(cache_folder)
 
 
+def print_info(cls_list, env_name=None):
+    """Print information for each element of input listed
+
+    Parameters
+    ----------
+    cls_list : list
+        List of objects (solvers or datasets) to print info from.
+    env_name : str | None
+        name of environment where to check for object availability.
+        If None or 'False', no check is made.
```

Comment:
```suggestion
        If None or False, no check is made.
```

Reply:
Done

Reply:
I think that I should let `'False'` because it is not a real boolean but a boolean in a string, c.f.:
```
@click.option('--env-name', 'env_name',
    metavar="<env_name>", type=str, default='False',
    shell_complete=complete_conda_envs,
    help="Additional checks for requirement availability in "
    "the conda environment named <env_name>.")
```